learn/intro/obol-splits.md

```diff
@@ -79,7 +89,7 @@ All source and target validators must be active with a balance greater than 32 e
 {% endhint %}
 
 {% hint style="info" %}
-Because the consolidation process is an asynchronous operation handled by the system contract, it is impossible to account for the deposited principal amount from the source validators. As a result, the principal of the consolidated validators is treated as a reward by the target ObolValidatorManager. This edge case will be addressed in a future update to the ObolValidatorManager smart contract.
+Because the consolidation process is an asynchronous operation handled by the system contract, it is impossible to account for the deposited principal amount from the source validators. As a result, the principal of the consolidated validators is treated as a reward by the target OVM. The target OVM can use `setAmountOfPrincipalStake()` function to adjust the current principal amount of stake.
```

Comment:
```suggestion
It is possible to permissionlessly consolidate a validator into (or out of) an OVM. This could result in the OVM's `amountOfPrincipalStake()` not accurately reflecting the true amount of stake on validators exiting to the OVM withdrawal address. This could result in more (or less) ether being treated as reward, and disbursed to the rewardRecipient address. The owner of the OVM or any address with the `SET_PRINCIPAL_ROLE` can update the amount of Ether treated as principal with the `setAmountOfPrincipalStake()` function. 
```